<ui version="4.0" >
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QtMPC Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="_3" >
     <property name="bottomMargin" >
      <number>18</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap" >
        <pixmap resource="../QtMPC.qrc" >:/images/icon.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>QtMPC Configuration</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <layout class="QGridLayout" name="_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_13" >
           <property name="text" >
            <string>MPD host:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3" >
          <widget class="QLineEdit" name="hostLineEdit" />
         </item>
         <item row="1" column="0" colspan="2" >
          <widget class="QLabel" name="label_14" >
           <property name="text" >
            <string>MPD port:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2" >
          <widget class="QSpinBox" name="portSpinBox" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>60000</number>
           </property>
           <property name="value" >
            <number>6600</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_15" >
           <property name="text" >
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3" >
          <widget class="QLineEdit" name="passwordLineEdit" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_16" >
           <property name="text" >
            <string>Crossfading (seconds)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3" >
          <widget class="QSpinBox" name="crossfading" >
           <property name="maximum" >
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3" >
          <spacer name="spacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>271</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="3" >
          <widget class="QCheckBox" name="systemTrayCheckBox" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Show in System tray</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3" >
          <spacer name="spacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>271</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="3" >
          <widget class="QCheckBox" name="systemTrayPopup" >
           <property name="text" >
            <string>Show Popup message</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="4" >
          <spacer name="spacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>596</width>
             <height>41</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0" >
          <widget class="QLabel" name="label_17" >
           <property name="text" >
            <string>Update Interval (miliseconds)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3" >
          <widget class="QSpinBox" name="updateInterval" >
           <property name="minimum" >
            <number>100</number>
           </property>
           <property name="maximum" >
            <number>4000</number>
           </property>
           <property name="singleStep" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>LastFM</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QCheckBox" name="lastFmCheckBox" >
         <property name="text" >
          <string>Enable lastFM</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_18" >
         <property name="text" >
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lastFmUserName" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_19" >
         <property name="text" >
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lastFmPassword" >
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtMPC.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>systemTrayCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>systemTrayPopup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>523</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>523</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastFmCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>lastFmPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>69</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel" >
     <x>369</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastFmCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>lastFmUserName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>69</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel" >
     <x>369</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
